tests/fault_remediation_test.go

```diff
@@ -36,7 +36,7 @@ func TestExistingCRPreventsNewCreation(t *testing.T) {
 		return newCtx
 	})
 
-	feature.Assess("existing CR prevents duplicate creation", func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
+	feature.Assess("new CR should be created on the same node after success", func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
```

Comment:
should we not not send a healthy event and verify that the new CR is not created? Are we checking state clearance with this test? If yes, can we rename the function?